### DISCUSSION TITLE
New address?

### DISCUSSION THREAD

**Discussion by mauver** [Chia-Network/chia-blockchain#2040] [Q&A] [answered]
when I clicked the new address, change the receive address. 

I need to change my plots..? what happened? 

When I farm the chia coin from block, what wallet address used? 

please describe this task..

**Comment by dorkmo** [comment, marked as answer]
the plots are locked to a certain address, you want to keep using the same address

**Reply by mauver** [reply]
can i use the first address? whenever i click the new address?

**Reply by dorkmo** [reply]
i usually just enter the 24 words again when i need to enter keys

**Reply by mauver** [reply]
thanks a lot

**Reply by adi100** [reply]
when I clicked the new address, How do I restore my First wallet address?

**Comment by adi100** [comment]
when I clicked the new address, How do I restore my First wallet address?
